bibtexparser/bwriter.py

```diff
@@ -14,6 +16,23 @@
 __all__ = ['BibTexWriter']
 
 
+class SortingStrategy(Enum):
+    ALPHABETICAL_ASC = auto()
+    ALPHABETICAL_DESC = auto()
+    PRESERVE = auto()
+
+
+def apply_sorting_strategy(strategy: SortingStrategy, items: Iterable[str]) -> Iterable[str]:
```

Comment:
I guess this method can be private (i.e., have a leading `_`)?

Reply:
True. Changed scope of method.